### ISSUE TITLE
Script installation is pointing on out of date TOMCAT Version

### ISSUE THREAD

**Issue by Cevenol38** [bigbluebutton/bbb-install#521]
Hello, 

When using the script to install BBB, Tomcat 8.5.39, which is out of date since 2019, is installed.
Is it possible to update your script or can we update by ourselves Tomcat to the latest version 8.5.79 ?

Thanks

**Comment by antobinary** [comment]
Hi @Cevenol38 

Just a bit of background:

We use 2 different Tomcat
On BigBlueButton 2.5+
1) Linux package for bbb-lti (optional package) and bbb-demo (optional package, must not be present on production)

```
# dpkg -l | grep tomcat
ii  libtomcat9-java                      9.0.31-1ubuntu0.2                  all          Apache Tomcat 9 - Servlet and JSP engine -- core libraries
ii  tomcat9                              9.0.31-1ubuntu0.2                  all          Apache Tomcat 9 - Servlet and JSP engine
ii  tomcat9-common                       9.0.31-1ubuntu0.2                  all          Apache Tomcat 9 - Servlet and JSP engine -- common files
```

and
2) Embedded Tomcat for bbb-web.

```
# ls /usr/share/bbb-web/WEB-INF/lib/tomcat-*
/usr/share/bbb-web/WEB-INF/lib/tomcat-annotations-api-9.0.62.jar
/usr/share/bbb-web/WEB-INF/lib/tomcat-embed-el-9.0.62.jar
/usr/share/bbb-web/WEB-INF/lib/tomcat-embed-websocket-9.0.62.jar
/usr/share/bbb-web/WEB-INF/lib/tomcat-embed-core-9.0.62.jar
/usr/share/bbb-web/WEB-INF/lib/tomcat-embed-logging-log4j-8.5.2.jar
```

Specifically for the embedded one, this is how we control the version: https://github.com/bigbluebutton/bigbluebutton/pull/14733/files



That said, which version of BigBlueButton are you using, and which script - bbb-install.sh vs bbb-install-2.5.sh ?

**Comment by Cevenol38** [comment]
Thanks, we are using bbb-install.sh 
BBB is mentionned as beta that is not compliant with our production environnement constraints.
What can we do ?

**Comment by ffdixon** [comment]
BigBlueButton 2.5 is currently in its 4th release candidate.  If you want to wait for the final release of 2.5, you may not need to wait too much longer.

**Comment by antobinary** [comment]
This should be resolved with the next 2.4 release (2.4.10), see https://github.com/bigbluebutton/bigbluebutton/pull/15202
